site/src/api/api.ts

```diff
@@ -4,6 +4,14 @@ import * as Types from "./types"
 import { WorkspaceBuildTransition } from "./types"
 import * as TypesGen from "./typesGenerated"
 
+// Always attach CSRF token to all requests
+const token = document.head.querySelector('meta[property="csrf-token"]')
+if (token) {
+  axios.defaults.headers.common["X-CSRF-TOKEN"] = token.getAttribute("content")!
+} else {
+  console.error("CSRF token not found: https://laravel.com/docs/csrf#csrf-x-csrf-token")
```

Comment:
What if we added something to our own docs about CSRF tokens not being found and how to troubleshoot?

Reply:
That was a copy pasta mistake :+1: